atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -136,6 +135,16 @@ def run(self):
         except KeyError:
             raise RuntimeError('missing label: {}'.format(version_label))
 
+        try:
+            _, release = labels.get_name_and_value(Labels.LABEL_TYPE_RELEASE)
+            if not self.append:
+                self.log.debug("release set explicitly so not incrementing")
+                if not is_scratch_build():
+                    self.check_build_existence_for_explicit_release(component, version, release)
+                return
+        except KeyError:
```

Comment:
Which method raises` KeyError`? This try-except block seems to be too wide to me, it should be minimalized with using try-except-else

Reply:
just the get release, I'll put other things outside of Try